build.fsx

```diff
@@ -0,0 +1,54 @@
+#r "packages/FAKE/tools/FakeLib.dll" // include Fake lib
+#r @"System.IO.Compression"
+#r @"System.IO.Compression.FileSystem"
+
+open System
+open System.Net
+open System.IO
+open System.IO.Compression
+open Fake 
+
+let sublimePath () = 
+  let UnixPaths = 
+      [  (Environment.GetEnvironmentVariable("HOME") + "/Library/Application Support/Sublime Text 3")
+         (Environment.GetEnvironmentVariable("HOME") + "/.config/sublime-text-3") ]
+  
+  let WindowsPaths = 
+      [ Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
+
+  let isWindows = (Path.DirectorySeparatorChar = '\\')
+  let searchPaths = if isWindows then WindowsPaths else UnixPaths
+  let directories = 
+     searchPaths 
+     |> List.filter Directory.Exists
+     
+  match directories.Length with
+     | 0 -> 
+         trace "No Sublime text 3 installation found" 
+         exit 1
+     | _ -> 
+         directories.Head
+
+Target "Clean" (fun _ ->
+    DeleteDirs ["bin"]
```

Comment:
Not sure how's this meant to be used. Bin still contains stuff to obtain fsac, doesn't it? If so, we need to sort out how to pull fsac into this project.